src/netex_france/exporter.rs

```diff
@@ -35,6 +35,27 @@ const NETEX_FRANCE_CALENDARS_FILENAME: &str = "calendriers.xml";
 const NETEX_FRANCE_LINES_FILENAME: &str = "lignes.xml";
 const NETEX_FRANCE_STOPS_FILENAME: &str = "arrets.xml";
 
+pub(in crate::netex_france) enum ObjectType {
+    DayType,
+    DayTypeAssignment,
+    Line,
+    Network,
+    UicOperatingPeriod,
+}
```

Comment:
:warning: This `enum` will need to be expanded with the below list if #544 is merged first:
- `Route`
- `ServiceJourneyPattern`
- `ScheduledStopPoint`
- `PassengerStopAssignment`
- `ServiceJourney`